pkg/util/tls/tls_test.go

```diff
@@ -0,0 +1,194 @@
+package tls
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const certPEM = `-----BEGIN CERTIFICATE-----
```

Comment:
This certificate has expired :) ... Should we check for that when loading it?

Reply:
I don't think we should, I think it should be for the server to decide if it's ok with the certificate